### PR TITLE
Fix exploit of bot auto confirming

### PR DESCRIPTION
**Pull request by chamburr**
The reaction that the bot adds to the message triggers the listener and the bot thinks that it confirmed.

### CODE REVIEW

**Comment by chamburr** [comment]
Also, a guy with id 340440443302969344 got $134,000 from this exploit so please remove their money!!!

File: classes/context.py [hunk 1]
```diff
@@ -48,6 +48,9 @@ def disp(self):
     ):
         user = user or self.author
         emojis = (emoji_no, emoji_yes)
+        
+        if user.id == self.bot.user.id:
+            return false
```
Comment (numbermaniac) [inline classes/context.py]: In Python the word is `False`, not `false`.

**Review by numbermaniac — CHANGES_REQUESTED** [review]

**Comment by numbermaniac** [comment]
Maybe a change should also be made somewhere around [here](https://github.com/Gelbpunkt/IdleRPG/blob/v4/cogs/trading/__init__.py#L239-L244) to prevent users offering items to any bot.

**Comment by Gelbpunkt** [comment]
Or just delete the Bot's account :P

**Comment by numbermaniac** [comment]
oh no

**Comment by chamburr** [comment]
God sorry I was dumb

**Comment by starspritechippy** [comment]
Not sure if you forgot about this but this bug still is in the bot.

**Pull request merged**